<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>mainWindow</class>
 <widget class="QMainWindow" name="mainWindow">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>700</width>
    <height>700</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>700</width>
    <height>700</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>700</width>
    <height>700</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Online Checker</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>icon.png</normaloff>icon.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="minimumSize">
    <size>
     <width>560</width>
     <height>700</height>
    </size>
   </property>
   <widget class="QTextEdit" name="te">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>492</width>
      <height>630</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Noto Sans KR</family>
     </font>
    </property>
   </widget>
   <widget class="QLabel" name="lb_not_text">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>600</y>
      <width>60</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>결석생:</string>
    </property>
   </widget>
   <widget class="QLabel" name="lb_not">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>600</y>
      <width>60</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="RB_1">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>190</y>
      <width>170</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>파일에서 불러오기(&amp;F)</string>
    </property>
    <property name="checked">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QComboBox" name="CB_1">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>230</y>
      <width>150</width>
      <height>32</height>
     </rect>
    </property>
   </widget>
   <widget class="QCheckBox" name="cb_num">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>530</y>
      <width>140</width>
      <height>35</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>번호만 표시(&amp;V)</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="RB_2">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>280</y>
      <width>125</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>값 생성하기(&amp;R)</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QSpinBox" name="sb_grade">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>310</y>
      <width>60</width>
      <height>33</height>
     </rect>
    </property>
    <property name="minimum">
     <number>1</number>
    </property>
    <property name="maximum">
     <number>3</number>
    </property>
   </widget>
   <widget class="QSpinBox" name="sb_class">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>350</y>
      <width>60</width>
      <height>33</height>
     </rect>
    </property>
    <property name="minimum">
     <number>1</number>
    </property>
    <property name="maximum">
     <number>10</number>
    </property>
   </widget>
   <widget class="QLabel" name="label_grade">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>310</y>
      <width>35</width>
      <height>29</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>학년</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_class">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>350</y>
      <width>18</width>
      <height>29</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>반</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="sb_last">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>390</y>
      <width>60</width>
      <height>33</height>
     </rect>
    </property>
    <property name="minimum">
     <number>1</number>
    </property>
    <property name="maximum">
     <number>40</number>
    </property>
    <property name="value">
     <number>30</number>
    </property>
   </widget>
   <widget class="QLabel" name="label_last">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>390</y>
      <width>58</width>
      <height>29</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>명</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="le_usr_num">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>500</y>
      <width>100</width>
      <height>30</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="lb_usr_number">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>500</y>
      <width>76</width>
      <height>29</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>자신 학번</string>
    </property>
   </widget>
   <widget class="QPushButton" name="bt_check">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>22</y>
      <width>180</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>&amp;Check</string>
    </property>
    <property name="checkable">
     <bool>false</bool>
    </property>
    <property name="checked">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="bt_delete">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>60</y>
      <width>180</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>&amp;Delete</string>
    </property>
   </widget>
   <widget class="QPushButton" name="bt_exit">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>98</y>
      <width>180</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>&amp;Quit</string>
    </property>
   </widget>
   <widget class="QComboBox" name="chat_txt">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>22</y>
      <width>455</width>
      <height>32</height>
     </rect>
    </property>
    <property name="currentText">
     <string notr="true"/>
    </property>
   </widget>
   <widget class="QPushButton" name="bt_chat_reload">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>22</y>
      <width>32</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>resource/view-refresh.svg</normaloff>resource/view-refresh.svg</iconset>
    </property>
    <property name="shortcut">
     <string>Ctrl+R</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="cb_save_list">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>420</y>
      <width>140</width>
      <height>35</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>명단 저장(&amp;S)</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
